lib/onboarding/widgets/about_myself.dart

```diff
@@ -217,12 +217,6 @@ class _InlineDropdownButton<T> extends StatelessWidget {
     final style = DefaultTextStyle.of(context).style;
 
     return DropdownButton(
-      selectedItemBuilder: (context) => items
-          .map((i) => DefaultTextStyle(
-                style: style,
-                child: i.child,
-              ))
-          .asList(),
```

Comment:
You should probably leave that code, because removing it was just a temporary fix.